src/com_tjqueue/admin/languages/en-GB/en-GB.com_tjqueue.ini

```diff
@@ -12,3 +12,5 @@ COM_TJQ_AWS_KEY="AWS Key"
 COM_TJQ_AWS_SECRET="AWS Secret"
 COM_TJQ_AWS_REGION="AWS Region"
 COM_TJQ_AWS_REGION_DESC="Region where sqs in located e.g <b>ap-south-1</b>"
+COM_TJQ_DEFAULT_LIMIT="Default Limit"
+COM_TJQ_DEFAULT_LIMIT_DESC="Queue records executed as per the limit. For if Limit is 5 then after executing the cron once, 5 records will get executed from the queue"
```

Comment:
@vijaykhollam `e.g` missing after For. 

* For e.g

Reply:
PR updated. Please check.